src/modules/autoDecomposeDeletionProposedRelationships/AutoDecomposeDeletionProposedRelationshipsModule.ts

```diff
@@ -0,0 +1,37 @@
+import { RelationshipChangedEvent, RelationshipStatus } from "@nmshd/runtime";
+import { ConnectorRuntimeModule, ConnectorRuntimeModuleConfiguration } from "../../ConnectorRuntimeModule";
+
+export interface AutoDecomposeDeletionProposedRelationshipsModuleConfiguration extends ConnectorRuntimeModuleConfiguration {}
+
+export default class AutoDecomposeDeletionProposedRelationshipsModule extends ConnectorRuntimeModule<AutoDecomposeDeletionProposedRelationshipsModuleConfiguration> {
+    private currentIdentity: string;
+
+    public init(): void {
+        // Nothing to do here
+    }
+
+    public async start(): Promise<void> {
+        const currentIdentityResult = await this.runtime.getServices().transportServices.account.getIdentityInfo();
```

Comment:
Should check for error, no?

Reply:
no, it will fail automatically when it failed.

Reply:
You mean accessing `.value` in case of an error will raise an exception?

Reply:
exactly!

Reply:
And as this module is only for internal / testing purposes we don't have to write special error messages. Also I don't see a reason why this call should fail :D